packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1197,10 +1276,31 @@ export const prepareVaultManagerKit = (
               ),
           );
 
+          const [{ userSeatPromise, deposited }, liquidationRecorderKits] =
+            await Promise.all([
```

Comment:
Should we use `Promise.allSettled` instead of `Promise.all`? The reason being, if `helper.makeLiquidationRecorderKits` somehow fails `Promise.all` is going reject all together and we won't be able to complete the liquidation operation. What are the possible chances of `helper.makeLiquidationRecorderKits` failing, I don't know. However, when we think about it, `helper.makeLiquidationRecorderKits` interacts with another vat by calling `makeChildNode`. So depending on `chainStorage` vat to work correctly when settling liquidations seems like an unnecessary dependecy.